### PR TITLE
add Dialog, fix sorting in empty table colon

### PR DESCRIPTION
**Pull request by VitalyiKhlivnyi**
create EventDetails
add color to evnt task tables
remove get tasks function from EventDetails component to EventTaskList
add update tasks list from DB, when user change task status

## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LEAR-33)
* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LEAR-5)

## Code reviewers

- [ ] @ashatna
- [ ] @ChristinaPastushchak
- [x] @dotiurin
- [x] @iryn34
- [ ] @tarassito
- [x] @VasilHamulyak
- [ ] @VitalyiKhlivnyi
- [ ] @YurijH

### Second Level Review

- [ ] @ushevtsiv
- [ ] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/static/src/containers/event/EventService.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ export const PostEventService = (data) => {
     axios.post(url, data);
 };
 
-export const deleteTeamService = (eventId, taskId) => {
-    const url =eventPath + eventId + '/task/'+taskId + '/';
+export const deleteTaskService = (eventId, taskId) => {
+    const url = eventPath + eventId + '/task/'+taskId + '/';
```
Comment (iryn34) [inline eventually/static/src/containers/event/EventService.js]: add whitespaces before and after +

File: eventually/static/src/containers/teamList/EditTeamDialog.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import Dialog from 'material-ui/Dialog';
 import FlatButton from 'material-ui/FlatButton';
 import TextField from 'material-ui/TextField';
-import FileUpload from 'src/containers';
+import {FileUpload} from 'src/containers';
 import {teamServicePut} from './teamService';
```
Comment (tarassito) [inline eventually/static/src/containers/teamList/EditTeamDialog.js:7]: add new line

File: eventually/static/src/containers/event/EventTaskList.js [hunk 3]
```diff
@@ -63,181 +83,149 @@ export default class EventTaskList extends React.Component {
...
+            />,
+        ];
+
+        let taskTable = [];
+        for(let counter = 0; counter < 3; counter++){
+            let strCounter = counter.toString() + this.state.tasksChangeStatus.toString();
+            const styleName = ['toDoBodyStyle', 'inProgressBodyStyle', 'doneBodyStyle'];
+            taskTable.push(
+                <td style={tdStyle[styleName[counter]]} key={strCounter}>
+                    <ul  id={counter} key={strCounter} type='none' ref={this.sortableGroupDecorator} style={ulStyle}>
```
Comment (iryn34) [inline eventually/static/src/containers/event/EventTaskList.js]: remove extra space after ul

File: eventually/static/src/containers/teamList/CreateTeamDialog.js [hunk 4]
```diff
@@ -124,7 +126,6 @@ export default class CreateTeamDialog extends React.Component {
                 'owner': getUserId(),
                 'members_id': this.state.values
```
Comment (iryn34) [inline eventually/static/src/containers/teamList/CreateTeamDialog.js:126]: use camelCase

File: eventually/static/src/containers/teamList/EditTeamDialog.js (hunk 2 above)
Comment (VitalyiKhlivnyi) [reply eventually/static/src/containers/teamList/EditTeamDialog.js:7]: Done

File: eventually/static/src/containers/teamList/CreateTeamDialog.js (hunk 4 above)
Comment (VitalyiKhlivnyi) [reply eventually/static/src/containers/teamList/CreateTeamDialog.js:126]: Done!

**Review by tarassito — DISMISSED** [review]

**Review by VasilHamulyak — DISMISSED** [review]

File: eventually/static/src/containers/event/EventDetails.js [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+                        eventId={this.state.eventId}
+                        teamId={this.state.teamId}
+                        style={TaskDialogStyle}/>
+                </div>)}
+            </div>
+        );
+    }
+}
+
+
```
Comment (dotiurin) [inline eventually/static/src/containers/event/EventDetails.js]: unnecessary blank line

File: eventually/static/src/containers/event/EventDetails.js [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+import EventTaskList from './EventTaskList';
+import TaskDialog from './CreateTaskDialog';
+
+
+const containerStyle = {
+    width: '99%',
+    margin: '0 auto',
+};
+
+const TaskDialogStyle = {
```
Comment (dotiurin) [inline eventually/static/src/containers/event/EventDetails.js]: use camaelCase

File: eventually/static/src/containers/event/EventDetails.js (hunk 5 above)
Comment (VitalyiKhlivnyi) [reply eventually/static/src/containers/event/EventDetails.js]: Done

File: eventually/static/src/containers/event/EventDetails.js (hunk 6 above)
Comment (VitalyiKhlivnyi) [reply eventually/static/src/containers/event/EventDetails.js]: Done

**Review by dotiurin — DISMISSED** [review]

**Review by iryn34 — DISMISSED** [review]

**Review by VasilHamulyak — DISMISSED** [review]

**Review by lhalam — DISMISSED** [review]

**Review by ashatna — APPROVED** [review]

**Review by lhalam — APPROVED** [review]

**Pull request merged**